mockwebserver/src/main/java/okhttp3/mockwebserver/Dispatcher.kt

```diff
@@ -31,7 +31,8 @@ abstract class Dispatcher {
    * request bodies.
    */
   open fun peek(): MockResponse {
-    return MockResponse().setSocketPolicy(SocketPolicy.KEEP_OPEN)
+    val mockResponse = MockResponse()
+    return mockResponse.apply { mockResponse.socketPolicy = SocketPolicy.KEEP_OPEN }
```

Comment:
InteliJ translated the deprecated fun to the code above.
Maybe it's better to do the following?
```
  open fun peek(): MockResponse = MockResponse().apply {
    this.socketPolicy = SocketPolicy.KEEP_OPEN
  }
```
or
```
  open fun peek(): MockResponse {
    return MockResponse().apply { this.socketPolicy = SocketPolicy.KEEP_OPEN }
  }
```

Reply:
the 2nd one!